docs/docsite/rst/network/dev_guide/developing_resource_modules_network.rst

```diff
@@ -558,69 +558,63 @@ The following example walks through the integration tests for the ``vyos.vyos.vy
 
 ``test/integration/targets/vyos_l3_interfaces/tests/cli/overridden.yaml``
 
-.. code-block:: yaml
+.. code-block:: yaml+jinja
 
   ---
-  - debug:
-   msg: START vyos_l3_interfaces merged integration tests on connection={{ ansible_connection
-     }}
+  - debug: null
+    msg: START vyos_l3_interfaces merged integration tests on connection={{ ansible_connection }}
 
   - import_tasks: _remove_config.yaml
 
   - block:
-
-   - import_tasks: _populate.yaml
-
-   - name: Overrides all device configuration with provided configuration
-     register: result
-     vyos.vyos.vyos_l3_interfaces: &id001
-       config:
-
-         - name: eth0
-           ipv4:
-
-             - address: dhcp
-
-         - name: eth1
-           ipv4:
-
-             - address: 192.0.2.15/24
-       state: overridden
-
-   - name: Assert that before dicts were correctly generated
-     assert:
-       that:
-         - "{{ populate | symmetric_difference(result['before']) |length == 0 }}"
-
-   - name: Assert that correct commands were generated
-     assert:
-       that:
-         - "{{ overridden['commands'] | symmetric_difference(result['commands'])\
-           \ |length == 0 }}"
-
-   - name: Assert that after dicts were correctly generated
-     assert:
-       that:
-         - "{{ overridden['after'] | symmetric_difference(result['after']) |length\
-           \ == 0 }}"
-
-   - name: Overrides all device configuration with provided configurations (IDEMPOTENT)
-     register: result
-     vyos.vyos.vyos_l3_interfaces: *id001
-
-   - name: Assert that the previous task was idempotent
-     assert:
-       that:
-         - result['changed'] == false
-
-   - name: Assert that before dicts were correctly generated
-     assert:
-       that:
-         - "{{ overridden['after'] | symmetric_difference(result['before']) |length\
-           \ == 0 }}"
+    - import_tasks: _populate.yaml
+    - name: Overrides all device configuration with provided configuration
+      register: result
+      vyos.vyos.vyos_l3_interfaces:
+        config:
+          - name: eth0
+            ipv4:
+              - address: dhcp
+          - name: eth1
+            ipv4:
+              - address: 192.0.2.15/24
+        state: overridden
+    - name: Assert that before dicts were correctly generated
+      assert:
+        that:
+          - "{{ populate | symmetric_difference(result['before']) | length == 0 }}"
+    - name: Assert that correct commands were generated
+      assert:
+        that:
+          - "{{ overridden['commands'] |
+            symmetric_difference(result['commands']) | length == 0 }}"
```

Comment:
That's a very weirdly indented and nested example. Mustache shouldn't be needed here. Plus there's better YAML syntax allowing to drop the quotes too, making the pipes look chained:
```suggestion
          - >-
            overridden['commands']
            | symmetric_difference(result['commands'])
            | length
            == 0
```

Reply:
The idea was for this PR to fix the syntax so it is valid, and then improve the formatting in a follow-up PR that's less critical than this one.